src/actions/create-environment/index.ts

```diff
@@ -24,13 +24,11 @@ export async function main(factory: RunnerFactory): Promise<void> {
         const createEnvironmentArgs = ['admin', 'create', '--name', envName, '--region', envRegion, '--type', envType, '--domain', domain];
         const result = await pac.run(createEnvironmentArgs);
         // HACK TODO: Need structured output from pac CLI to make parsing out of the resulting env URL more robust
-        const envUrl = result
-            .filter(l => l.length > 0)
-            .pop()
-            ?.trim()
-            .split(/\s+/)
-            .shift();
+        const resultArray = result.filter(l => l.length > 0).pop()?.trim().split(/\s+/);
```

Comment:
nit: that const name is not useful since it doesn't convey what it holds: the newly created environment's details. How about: newEnvDetailColumns or similar?
Also, pls bring back the multi-line formatting, that makes chained expressions easier on the human reader